FlightPlanner/Controllers/AdminApiController.cs

```diff
@@ -0,0 +1,82 @@
+﻿using AutoMapper;
+using FlightPlanner.Api.Dtos;
+using FlightPlanner.Core.Models;
+using FlightPlanner.Core.Services;
+using FluentValidation;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
```

Comment:
Kāpēc Tu lock izmanto kontrolierī nevis servisā? Labāk lai lock ir servisā, tad ja tu to pašu servisu izmantosi citur, tad nebūs divreiz lock jālieto. Un kontrolierī vispār nav ieteicams lietot lock.